src/procedures/__tests__/CreateEtherDividendDistribution.ts

```diff
@@ -48,6 +57,14 @@ describe('CreateEtherDividendDistribution', () => {
     );
     tokenFactoryMockStub = tokenFactoryMock.mock('getSecurityTokenInstanceFromTicker', {});
```

Comment:
This stub doesn't seem to be used anywhere, so I don't see the need to assign it to a variable